articles/virtual-machines/virtual-machines-linux-extensions-customscript.md

```diff
@@ -136,8 +136,8 @@ The Azure Custom Script Extension can be run at Virtual Machine deployment time
 
 ```json
 {
-    "name": "scriptextensiondemo",
-    "type": "extensions",
+    "name": "[concat(parameters('scriptextensiondemoName'), 'scriptextensiondemo')]",
```

Comment:
I'm not sure if I follow this. As I understand the name value here, this is the name that will be given to the instance of the extension. what is the logic in building the name from a parameter? 

The current example has a 'hard coded' name for simplicity. Is there an issue with this?